installation/Makefile

```diff
@@ -11,7 +11,7 @@ ifeq ($(OS),Windows_NT)
 else
 	UNAME_S := $(shell uname -s)
 	ifeq ($(UNAME_S),Linux)
-		DOCKER_COMPOSE = docker-compose
+		DOCKER_COMPOSE = docker compose
```

Comment:
Please confirm this is Ok in linux?

Reply:
using docker-compose gives me errors, because the version that I install uses docker compose